Makefile

```diff
@@ -58,15 +34,4 @@ css:
 clean:
 	# Remove files not in source control
 	find . -type f -name "*.pyc" -delete
-	rm -rf nosetests.xml coverage.xml htmlcov *.egg-info *.pdf dist violations.txt
-
-preflight: lint
-    # Bare minimum of tests to run before pushing to master
-	./runtests.py
-
-todo:
-	# Look for areas of the code that need updating when some event has taken place (like 
-	# Oscar dropping support for a Django version)
-	-grep -rnH TODO *.txt
-	-grep -rnH TODO src/oscar/apps/
-	-grep -rnH "django.VERSION" src/oscar/apps
+	rm -rf nosetests.xml coverage.xml htmlcov *.egg-info *.pdf dist violations.txt
```

Comment:
missing newline at end of file

sollte eigentlich im editor eingestellt werden können.

Reply:
Fixed